inputFiles/thermalSinglePhaseFlowFractures/egsCollab_thermalFlow/egsCollab_thermalFlow_base.xml

```diff
@@ -80,7 +80,8 @@
 
      <SolidInternalEnergy
        name="rockInternalEnergy"
-       volumetricHeatCapacity="2.3345e6"
+       referenceVolumetricHeatCapacity="2.3345e6"
+       dVolumetricHeatCapacity_dTemperature="1e6"
```

Comment:
dVolumetricHeatCapacity_dTemperature for this case should be zero (default value)?
```suggestion

```

Reply:
Well captured. Done